### PR TITLE
Fixing highp issue - Batch Tables

### LINKED ISSUE #8311: Use highp sampler2D when using float textures for tables
https://github.com/AnalyticalGraphicsInc/cesium/issues/6739 observed that Cesium's float batch tables didn't seem to be working at high enough precision on some devices.
From https://groups.google.com/forum/#!topic/webgl-dev-list/aFmFGeyRVRQ, devices are free to select the precision of `sampler2D` when it isn't explicitly set.
We should update our batch table code to use `uniform highp sampler2D` when backing the batch table with float textures.

As part of this, please roll back changes from https://github.com/AnalyticalGraphicsInc/cesium/pull/7421 and test on a device that shows red on https://likangning93.github.io/webgl-examples/tests/floatTexturePrecision/ but no red on https://likangning93.github.io/webgl-examples/tests/floatTexturePrecisionHighpSampler/.

Good candidates as of this writing are the iPhone 7 or newer and the first generation iPad Pro or newer.

### PR DESCRIPTION
**Pull request by YoussefV**
Fixes #8311

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @YoussefV!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/highp-batch-tables-fix/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by mramato** [comment]
@likangning93 mentions in the linked issue that we should roll back #7421, is that still the case?

**Comment by YoussefV** [comment]
> @likangning93 mentions in the linked issue that we should roll back #7421, is that still the case?

@mramato Just addressed this in the latest commit. The answer turned out to be yes, but not exactly. Because #7421 was fairly old, it needed careful rolling back. Latest push should have fixed it, I needed to only remove code that was related to the high precision issue.

**Comment by likangning93** [comment]
Screenshot_2020-04-28_18-00-23

:clap::clap::clap:

File: CHANGES.md [hunk 1]
```diff
@@ -205,6 +205,8 @@ var viewer = new Viewer("cesiumContainer", {
 
 ##### Fixes :wrench:
 
+<<<<<<< HEAD
```
Comment (mramato) [inline CHANGES.md:208]: Looks like there were some merge issues with this file.

Comment (YoussefV) [reply CHANGES.md]: woops! Fixed :)

File: Specs/Scene/ShadowVolumeAppearanceSpec.js [hunk 2]
```diff
@@ -335,188 +272,13 @@ describe("Scene/ShadowVolumeAppearance", function () {
     });
   });
 
-  describe("floating point textures unreliable", function () {
```
[the remark this answers is not in the document]
Comment (YoussefV) [reply Specs/Scene/ShadowVolumeAppearanceSpec.js:338]: removed the describe block

**Pull request merged**